pkg/crc/machine/bundle/repository.go

```diff
@@ -0,0 +1,78 @@
+package bundle
+
```

Comment:
Is this code movement only? are there expected  behaviour changes? Would be useful to know in the commit log :)

Reply:
Only move, added comment.